### DISCUSSION TITLE
Can't a node be in two combo?

### DISCUSSION THREAD

**Discussion by Time-coding** [antvis/G6#6668] [Q&A]
![image](https://github.com/user-attachments/assets/360532ee-6eb2-4fe4-9151-514154ab61b5)
NodeData的combo属性可以是一个string[]吗。一个节点请允许出现在两个combo中

**Comment by petercat-assistant** [comment]
In the current version of G6, a node is typically designed to belong to only one combo. The `combo` attribute in `NodeData` is generally a single string that represents the ID of the combo to which the node belongs. Allowing a node to be part of multiple combos would require a different structure and logic, as it is not the default behavior supported by G6.

If you want to achieve this functionality, you might need to implement a custom solution or modify the existing logic to handle nodes belonging to multiple combos. This could involve managing the relationships and rendering logic manually.

If this is a feature you would like to see in future versions of G6, you might consider proposing it as a feature request to the G6 development team.

If you have any more questions or need further assistance, feel free to ask!

<details>
<summary>🪧 Tips</summary>
For further assistance, please describe your question in the comments and @petercat-assistant to start a conversation with me.
</details>

**Comment by liaodalin19903** [comment]
I think plugin `Hull ` could meets your needs.
